### DISCUSSION TITLE
Longhorn as CSI for the postgres-operator?

### DISCUSSION THREAD

**Discussion by ALGOD-SA** [zalando/postgres-operator#1987] [Q&A]
did anyone try longhorn as CSI for the postgres-operator? sounds interesting in theory, but don't know if they overlap on different levels. For instance, they both handle backups.

**Comment by djcenox** [comment]
We are using it, but only for volume management not for backups. Anyway we thought that would be more "safe" to let postgresql backup itself instead snapshotting the underling block device.
